src/Command/GenerateStatisticsCommand.php

```diff
@@ -16,9 +16,9 @@ final class GenerateStatisticsCommand extends Command
      */
     private $analyseStatistics;
 
-    public function __construct(AnalyseStatistics $analyseStatistics)
+    public function __construct(AnalyseStatistics $deleteOldDownloadLogs)
```

Comment:
Could you revert this change?

Reply:
hah, yes must've missed that in PhpStorm's refactoring :)